ibm-messaging-mq-cloud-showcase-app/backend/AWS-showcase-backend/msms/message-session-manager.js

```diff
@@ -90,6 +90,58 @@ class MQClient {
     this.myID = uuidv4();    
   }
 
+  /**
+ * Checks if a reply to queue exists or not
+ * @param {Object} replyToMsg The reply to queue value
+ * @param {Object} appId The current session id.
+ * @returns {Promise<any>} Returns a promise resolving to true if the reply to queue exists, else resolves with a false.
+ */
+  checkQueueExists(replyToMsg, appId) {
```

Comment:
This method checks if a queue exists, it is not specific to a reply queue. I suggest changing the name, and comments, to make it less specific.